corehq/apps/auditcare/couch_to_sql.py

```diff
@@ -41,7 +41,7 @@ def copy_events_to_sql(start_time, end_time):
     util.log_batch_start(key)
     break_query = False
     last_doc_id = None
-    count = util.get_existing_count(key)
+    count, other_doc_count = util.get_existing_count(key)
```

Comment:
nit: looks like this value has different names in different places (`other_doc_count` vs `other_doc_type_count`). It would make the code easier to read if the same name were used throughout.